src/main/scala/nodes/images/Pooler.scala

```diff
@@ -67,3 +67,21 @@ class Pooler(
     ChannelMajorArrayVectorizedImage(patch, ImageMetadata(numPoolsX, numPoolsY, numChannels))
   }
 }
+
+object Pooler {
+
+  def sumPooler(in: DenseVector[Double]): Double = {
```

Comment:
Its slower - and constantly showed up in `jstack` while I was running experiments. To get a sense of how much slower here is a unscientific benchmark I ran on my laptop
```
var sum=0.0; var sumTime=0.0; 
for (i <- 0 until 10000) { 
  val a = DenseVector.rand(4096);
  val begin = System.nanoTime(); 
  sum += sumPooler(a);  // Replace this with a.sum for the Breeze one
  sumTime += (System.nanoTime() - begin)/1000.0 
};
```
Avg sumTime for breeze: 163.9452 microseconds
Avg sumTime for while loop: 21.4573 microseconds

At length=8192
Avg sumTime for breeze: 328.2782 microseconds
Avg sumTime for while loop: 40.1276 microseconds

At length=81920
Avg sumTime for breeze: 3351.0207 microseconds
Avg sumTime for while loop: 412.5542 microseconds

So this 8x overhead seems to be consistent as we increase the vector length.